### ISSUE TITLE
Allow unidentified issues to be connected to legacy issues

### ISSUE THREAD

**Issue by leikkisa** [department-of-veterans-affairs/caseflow#12694]
## User or job story
As an intake user, when I add an unidentified request issue, I should be able to connect legacy issues to it for opt-in.

## Acceptance criteria
- [ ] Please put this work behind the feature toggle: verify_unidentified_issue
- [ ] When adding an unidentified issue, trigger the legacy opt-in modal after the user adds an unidentified issue
- [ ] This should happen in the same order it is happening elsewhere, after correction type and before untimely exemption (as usual)
- [ ] The unidentified issue should get the same legacy issue eligibility checks (as usual)
- [ ] The legacy issue(s) should be saved as connected to the unidentified issue (as usual)
- [ ] Opt-ins should be created for the unidentified issue if is not ineligible (as usual)
- [ ] Include screenshot in the GH issue of the add issues page with an unidentified issue with a legacy issue opt-in connected to it

## Release notes

Unidentified issues can be connected to legacy issue opt-ins.  This is being added to support the use of unidentified issues to handle missing ratings.

### Out of scope
Being able to process multiple legacy opt-ins.  That is being handled in other tickets, with the final one being #12693.  This work can happen in parallel, however it would be good to pay attention to the current status of that work for merge conflicts or other impacts.

### Background/context
Historically we tried to force users to remove unidentified issues, and did not allow them to continue to be decided on.  If we do allow them to continue to be processed and decided, then we need to allow them to also opt-in legacy issues if appropriate.

### Technical notes

On the front-end, the AddIssueManager, AddedIssue, and LegacyOptInModal may be good files to start to look at.

I am marking some AC with `(as usual)` to indicate that code may exist that does this for unidentified issues already, or for other types of issues where you can see the pattern

**Comment by Sjones352** [comment]
Connected to legcay issue
<img width="1143" alt="Screen Shot 2019-12-31 at 12 42 37 AM" src="https://user-images.githubusercontent.com/23080951/71611415-95511900-2b66-11ea-90ec-5defea41218a.png">

<img width="1013" alt="Screen Shot 2019-12-31 at 12 38 59 PM" src="https://user-images.githubusercontent.com/23080951/71630012-bc433580-2bce-11ea-8ff3-b20ff53582d5.png">
